.svlint.toml

```diff
@@ -0,0 +1,42 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Rules for svlint, a SystemVerilog linter commonly used in editors.
+# The configuration matches the lowRISC SystemVerilog style guide at
+# https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md.
+# See https://github.com/dalance/svlint/blob/master/RULES.md for a list of rules.
+
+[option]
+exclude_paths = ["build.*", "sw/.*", ".sv.tpl$", "vendor/*"]
```

Comment:
exclude_paths is a regular expression, where "any character" is `.*`. So the last item should be `"vendor/.*`.